### PR TITLE
Inherit JDK16 testing platforms for JDK17

### PR DESCRIPTION
**Pull request by M-Davies**
Closes: #2338 
Signed-off-by: Morgan Davies <morgandavies2020@gmail.com>

### CODE REVIEW

**Comment by M-Davies** [comment]
run tests

**Review by smlambert — APPROVED** [review]

**Review by aahlenst — APPROVED** [review]
Thanks!

File: pipelines/jobs/configurations/jdk17_pipeline_config.groovy [hunk 1]
```diff
@@ -106,10 +97,10 @@ class Config17 {
                 os                  : 'windows',
                 arch                : 'x86-32',
                 additionalNodeLabels: 'win2012&&vs2017',
-                buildArgs : [
+                buildArgs           : [
                         hotspot : '--jvm-variant client,server'
                 ],
-                test                : ['sanity.openjdk']
+                test                 : 'default'
```
Comment (sxa) [inline pipelines/jobs/configurations/jdk17_pipeline_config.groovy:103]: Despite a lot of white space changes in here, this one seems to have an extra space (assuming the goal was to line up all the `:` characters)

**Review by sxa — APPROVED** [review]
Whitespace nit, but otherwise ok (although we now have two PRs doing this)

File: pipelines/jobs/configurations/jdk17_pipeline_config.groovy (hunk 1 above)
Comment (M-Davies) [reply pipelines/jobs/configurations/jdk17_pipeline_config.groovy]: Fixed

**Review by andrew-m-leonard — APPROVED** [review]
lgtm

**Comment by M-Davies** [comment]
I would also like to congratulate me, myself and I for being the 2000th commit to the master branch of this repo 😊

**Pull request merged**